.github/workflows/android.yml

```
@@ -43,10 +43,16 @@ jobs:
       run: |
         echo kakao_native_app_key="$KAKAO_NATIVE_APP_KEY" > ./local.properties
 
+    # firebase위한 google-services.json 파일 생성
+    - name: Create google-services.json
+      env:
+        GOOGLE_SERVICE_JSON: ${{ secrets.GOOGLE_SERVICE_JSON }}
+      run: echo "GOOGLE_SERVICE_JSON" > app/google-services.json
```

```suggestion
      run: echo "$GOOGLE_SERVICE_JSON" > ./app/google-services.json
```
나 CI는 잘 모르긴 하는데.. 
1) 이거 경로 앞에 `./` 안붙여도 괜찮은거지..?
2) GOOGLE_SERVICE_JSON 앞에 $ 안붙여도 괜찮은거지..?

일단 어푸 할태니 괜찮은거 확인되면 머지해줘!!